WikiWatershed/model-my-watershed
# Add elevation layer to layerpicker

## Overview

* Add the painted elevation layer to the app
* 6ee745b shouldn't change any terrain results, but I will be deleting the old layer when this is on staging
* 6933093 fixes https://rollbar.com/Azavea/MMW/items/277/

Connects #2629 

### Demo

<img width="294" alt="screen shot 2018-03-02 at 11 01 22 am" src="https://user-images.githubusercontent.com/7633670/36911771-977881ae-1e12-11e8-8fb3-1ab510b08fac.png">

![screen shot 2018-03-01 at 2 51 11 pm](https://user-images.githubusercontent.com/7633670/36911774-97990fdc-1e12-11e8-94d8-a7a2fafb9c90.png)

![screen shot 2018-03-01 at 2 50 10 pm](https://user-images.githubusercontent.com/7633670/36911776-97ae5efa-1e12-11e8-89ec-161cdd82ab8a.png)
![screen shot 2018-03-01 at 2 49 52 pm](https://user-images.githubusercontent.com/7633670/36911777-97b87eb2-1e12-11e8-98ff-b373c777cb54.png)

## Testing Instructions

 * Pull and `bundle`
 * Restart celery, if necessary
 * Open the coverage grids in the layer selector and turn on "Elevation". Behold its beauty
 * Behold its layer legend
 * With the layer on, draw a shape where the layer does not exist (eg lake michigan or the florida keys). Confirm the terrain tab does not display an error
 * Draw a shape where the layer has some values and confirm the results match the painted version (PA should be <20m is, Rockies should be >2000m)
